static/js/message_edit.js

```diff
@@ -700,6 +721,7 @@ export function save_message_row_edit(row) {
     const msg_list = message_lists.current;
     let message_id = rows.id(row);
     const message = message_lists.current.get(message_id);
+    const can_edit_stream = message.is_stream && page_params.is_admin;
```

Comment:
We have added a setting to control who can move messges between streams, and it does not restrict to admins only now. You need to use `settings_data.user_can_move_messages_between_streams` instead of `page_params.is_admin`.

Reply:
Awesome! Thanks :)